WordPress/Classes/ViewRelated/Reader/Manage/ReaderTagsTableViewController.swift

```diff
@@ -75,16 +79,17 @@ extension ReaderTagsTableViewController {
         let placeholder = NSLocalizedString("Add any tag", comment: "Placeholder text. A call to action for the user to type any tag to which they would like to subscribe.")
         let controller = SettingsTextViewController(text: nil, placeholder: placeholder, hint: nil)
         controller.title = NSLocalizedString("Add a Tag", comment: "Title of a feature to add a new tag to the tags subscribed by the user.")
-        controller.onValueChanged = { value in
-            if value.trim().count > 0 {
-                self.followTagNamed(value.trim())
+        controller.onValueChanged = { [weak self] value in
+            let tagName = value.trim()
+            if tagName.isEmpty() {
```

Comment:
Shouldn't this be `if !tagName.isEmpty()` (since the original was `if value.trim().count > 0`)?

Reply:
Yes, just changed this and made a few API modifications.